src/db.rs

```diff
@@ -89,6 +90,7 @@ impl Database {
     async fn initialize(&mut self) -> Result<()> {
         let db = self.0.get()?;
 
+        // Create categories table
```

Comment:
Appreciate the comments, I think it would be nice to move all these init queries into a new file at some point instead of having everything in one big DB file (maybe also submethods for a better overview).

Reply:
Yeah, I'll do a refactor pass, the file is becoming a bit long.